src/modules/dataGroups.js

```diff
@@ -37,40 +35,47 @@ class DataGroups extends React.PureComponent {
     }
 
     resizeWindow() {
-        const { columns } = this.props;
-        const columnLength = columns.length;
-        let columnCount = Math.floor(this.dataGroups.offsetWidth / 298);
-
-        if (columnCount < 1) {
-            columnCount = 1;
-        } else if (columnCount > columnLength) {
-            columnCount = columnLength;
-        }
-
-        this.reorderColumns(columnCount);
+        this.reorderColumns();
     }
 
-    reorderColumns(columnCount) {
+    reorderColumns() {
         const { columns } = this.props;
-        const newColumns = [];
+        const blockSize = this.dataGroups.offsetWidth;
+        const columnMinWidth = 300;
+        const columnsLength = columns.length;
+        let amountOfColumns = Math.floor(blockSize / columnMinWidth);
```

Comment:
`numberOfColumns` reads slightly better here (i.e. is more grammatically correct) than `amountOfColoumns`.  The difference between "number of" and "amount of" is subtle, having to do with whether the quantity being described is easily enumerable (countable), or if it is really, really large; see https://writingexplained.org/amount-vs-number-difference

Reply:
Fixed.